### ISSUE TITLE
Incorrect heading level is provided for "WebScout" and "Recommendation" headings.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/sarif-web-component#29]
### Actual:
NVDA is announcing the incorrect heading levels for the headings "WebScout" and "Recommendation". NVDA is announcing "heading level 1" for "WebScout" and "heading level 2" for "Recommendation".

**Similar issue is also observed with Chromium Edge + Narrator.**

### Expected:
NVDA should not announce the correct heading levels for the headings "WebScout" and "Recommendation". NVDA should announce "heading level 2" for "WebScout" and "heading level 3" for "Recommendation".

**Comment by jeffersonking** [comment]
@shreyashi227 Please provide a screenshot or the SARIF Log you are viewing.

While the string "WebScout" is likely the tool name displayed in the card title, the string "Recommendation" is not meaningful less we have more context.

**Comment by shreyashi227** [comment]
@jeffersonking 
Could you please provide us more information here for the above comment

**Comment by jeffersonking** [comment]
"WebScout" should be fixed in v0.5 (2413b17b8cf2f5a11cf4238df5418f00eddd43a1).

@shreyashi227 Can you show me where you see the text "Recommendation"?

**Comment by shreyashi227** [comment]
@jeffersonking 
"Recommendation" heading is present on each row of the grid under the "Details" column header
